utils/certs/gen_telemetry_admin_certificate.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
```

Comment:
We've already talked a bit about these cert generation scripts... I'm fine with leaving them in the short term, but I think we should switch to something more similar to what real DAOS admins will be using for 2.8+.

I can live with the self-signed cert for our internal purposes, but I don't think it's a good idea to encourage admins to use these scripts. Admins should use whatever CA their organization has approved to get certs for each machine. If there is already an HTTPS cert for the domain, they don't need a special one for DAOS telemetry, as long as daos_server/daos_agent has access to the key file.

If there is not already a certificate for the domain name, using a free tool like [Let's Encrypt](https://letsencrypt.org/) is better than using a self-signed cert. It also frees us up from trying to support customers using these scripts, which they will inevitably do.

Reply:
Same as above will check the tool and update the code.